File: java-apps/kq-common/src/main/java/com/kafkaquest/kq/common/util/models/events/enums/EventStatus.java
package com.kafkaquest.kq.common.util.models.events.enums;

public enum EventStatus {

    GAME_ENDED(0),
    GAME_PAUSED(1),
    GAME_RESUMED(2),
    OTHER(3);

    private final int eventStatusNumber;

    EventStatus(int eventStatusNumber) {
        this.eventStatusNumber = eventStatusNumber;
    }

    public int getEventStatusNumber() {
        return eventStatusNumber;
    }

}

File: kq-projection-dotnet/kq-projection/Common/Enums/EventStatus.cs
namespace KQ_Projection.Common.Enums;

using System.Runtime.Serialization;
using System.Text.Json.Serialization;

[JsonConverter(typeof(JsonStringEnumConverter))]
public enum EventStatus
{
    [EnumMember(Value = "GAME_ENDED")]
    GAME_ENDED = 0,

    [EnumMember(Value = "GAME_PAUSED")]
    GAME_PAUSED = 1,

    [EnumMember(Value = "GAME_RESUMED")]
    GAME_RESUMED = 2,

    [EnumMember(Value = "OTHER")]
    OTHER = 3
}
